### DISCUSSION TITLE
Error:  ModuleNotFoundError: No module named 'spacy.training'

### DISCUSSION THREAD

**Discussion by yosiasz** [explosion/spaCy#6623] [Language Support]
Greetings,

I get this error when running

```
py.test tests --model en_core_web_sm --lang en --has-parser --has-tagger --has-ner
```

I am following the steps here

https://github.com/explosion/spacy-models/tree/master/tests
What am I missing?

Thanks

**Comment by adrianeboyd** [comment]
The `spacy-models` tests have been updated for v3 already. To run the v2 tests, check out the last v2-compatible version at this commit:

https://github.com/explosion/spacy-models/commit/8222f7aa4e24acf76f11b61cca42b6d9d34f23ef

We should add a tag for v2 and update the README so this is documented better.
